MODULE.bazel

```diff
@@ -2,7 +2,16 @@
 # Exceptions. See /LICENSE for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-"""Bazel modules."""
+"""Bazel modules.
+
+If `MODULE.bazel.lock` changes locally, it means the host platform is not part
+of the GitHub supported set. Create a new PR with the result of
+`bazel mod deps`.
```

Comment:
This isn't enough information for me to be able to follow these instructions. What would I do with the result of `bazel mod deps`?

Reply:
Rephrased